playwright/Customizations/OpenSCAP.spec.ts

```diff
@@ -171,9 +171,67 @@ test('Create a blueprint with OpenSCAP customization', async ({
       .click();
   });
 
-  // This is for hosted service only as these features are not available in cockpit plugin
   await test.step('Export BP', async () => {
-    await exportBlueprint(page, blueprintName);
+    await exportBlueprint(frame, blueprintName, JSON.stringify(
```

Comment:
for openscap we might actually not want to do this, as the profile could change (and thus the tests would suddenly break without having changed anything).